ros2action/package.xml

```diff
@@ -27,8 +27,11 @@
   <test_depend>ament_flake8</test_depend>
   <test_depend>ament_pep257</test_depend>
   <test_depend>ament_xmllint</test_depend>
+  <test_depend>launch</test_depend>
+  <test_depend>launch_ros</test_depend>
```

Comment:
Technically, I don't think this package directly depends on `launch_ros`.

Reply:
Fixed in 6f5a060